### PR TITLE
Move travis-specific bash files to ~travis/.bash_profile.d/

### PR DESCRIPTION
**Pull request by meatballhat**
Addresses #862

### CODE REVIEW

**Comment by meatballhat** [comment]
Testing this out in https://travis-ci.org/travis-infrastructure/packer-build/builds/253683489

File: cookbooks/travis_build_environment/recipes/ci_user.rb [hunk 1]
```diff
@@ -31,12 +35,12 @@
...
-  { name: "#{node['travis_build_environment']['home']}/builds", perms: 0o755 },
-  { name: "#{node['travis_build_environment']['home']}/.m2" },
-  { name: "#{node['travis_build_environment']['home']}/gopath" },
-  { name: "#{node['travis_build_environment']['home']}/gopath/bin" }
+  { name: home },
+  { name: home.join('.m2') },
+  { name: home.join('.ssh') },
+  { name: home.join('builds'), perms: 0o755 },
+  { name: home.join('gopath') },
+  { name: home.join('gopath/bin') }
```
Comment (edmorley) [inline cookbooks/travis_build_environment/recipes/ci_user.rb]: I wonder if I should move the `$HOME/bin` directory creation here from the packer build scripts?

Comment (meatballhat) [inline cookbooks/travis_build_environment/recipes/ci_user.rb]: Why not both? 😁

**Comment by meatballhat** [comment]
Looking good 👌 https://travis-ci.org/travis-infrastructure/packer-build/builds/253694406

**Pull request merged**